app/src/main/java/com/android/sample/ui/settings/SettingsScreen.kt

```diff
@@ -0,0 +1,108 @@
+package com.android.sample.ui.settings
+
+import android.widget.Toast
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.ArrowBack
+import androidx.compose.material.icons.outlined.Delete
+import androidx.compose.material.icons.outlined.ExitToApp
+import androidx.compose.material3.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.android.sample.ui.navigation.NavigationActions
+import com.google.firebase.auth.FirebaseAuth
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun SettingsScreen(navigationActions: NavigationActions) {
+  val context = LocalContext.current
+
+  Scaffold(
+      topBar = {
+        TopAppBar(
+            title = { Text("Settings", fontSize = 20.sp) },
+            navigationIcon = {
+              IconButton(onClick = { navigationActions.goBack() }) {
+                Icon(imageVector = Icons.Outlined.ArrowBack, contentDescription = "Back")
+              }
```

Comment:
Suggestion: Since the TopAppBar is very similar in all screens you could extract a common composable. In this case it might a bit too small to really be useful